sensors/ev3_uart_sensor_defs.c

```diff
@@ -86,25 +86,67 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 			},
 			[3] = {
 				/**
-				 * @description: Raw Reflected - sets LED color to red
-				 * @value0: ??? (0 to 1020???)
-				 * @value1: ??? (0 to 1020???)
+				 * .. [#lego-ev3-color-mode3-values] This mode
+				 *    provides direct access to the measurements
+				 *    done by the ADC inside the sensor. It
+				 *    works by continuously flashing the red LED
+				 *    on and off and measuring the current
+				 *    flowing through a bypass capacitor. It is
+				 *    not possible to measure true ambient light
+				 *    this way, but the LED off measurement
+				 *    works as a reference for the LED on
+				 *    measurement. Higher photodiode
+				 *    illumination will cause lower ADC
+				 *    measurements., so to obtain reflected
+				 *    light intensity similar to
+				 *    ``COL-REFLECT``, it is necessary to
```

Comment:
I have found double backticks in the surrounding documentation (for example around line 313):
```
 * .. [#lego-ev3-gyro-mode0-value0-overflow] If
 *    you spin around too many times in ``GYRO-ANG``
 *    or ``GYRO-G&A`` mode, it will get stuck at
 *    32767 or overflow through -32768 depending
 *    on when the sensor was manufactured.
```